requirements/development.txt

```diff
@@ -4,24 +4,22 @@
 #
 #    make upgrade
 #
+appdirs==1.4.4            # via fissix
 astroid==2.4.2            # via -r requirements/testing.txt, pylint
 attrs==20.3.0             # via -r requirements/testing.txt, pytest
-importlib-metadata==2.0.0  # via -r requirements/testing.txt, pluggy, pytest
+fissix==20.8.0            # via modernize
 iniconfig==1.1.1          # via -r requirements/testing.txt, pytest
 isort==4.3.21             # via -r requirements/testing.txt, pylint
 lazy-object-proxy==1.4.3  # via -r requirements/testing.txt, astroid
 mccabe==0.6.1             # via -r requirements/testing.txt, pylint
-modernize==0.7            # via -r requirements/development.in
-packaging==20.4           # via -r requirements/testing.txt, pytest
-pathlib2==2.3.5           # via -r requirements/testing.txt, pytest
+modernize==0.8.0          # via -r requirements/development.in
```

Comment:
Do we actually need the modernize dependency?  I think it's no longer needed now that we've finished the Python 3 upgrade.

Reply:
Removed the `modernize` dependency.